test/test-nagoya-university-conversation-corpus.rb

```diff
@@ -0,0 +1,128 @@
+class NagoyaUniversityConversationCorpusTest < Test::Unit::TestCase
+  def setup
+    @dataset = Datasets::NagoyaUniversityConversationCorpus.new
+  end
+
+  test('#each_sentences') do
```

Comment:
Could you use the following style?

```ruby
sub_test_case("#each") do
  test("#sentences") do
    # ...
  end

  test("#participants") do
    # ...
  end

  test("others") do
     # ...
  end
end
```

We use `#XXX` for instance method name because the notation is widely used in Ruby documents.
So we don't want to use `#XXX` for non-instance method such as `#each_sentences`.

Reply:
Thanks for the explanation. I understand.